network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/custom_hero.html

```diff
@@ -5,13 +5,17 @@
 {% with control_btn_classes="ml-auto tw-opacity-80 hover:tw-opacity-100 tw-hidden" full_screen_content_classes="tw-absolute tw-min-w-full tw-min-h-full tw-max-w-none tw-top-0" two_col_content_classes="tw-h-full tw-w-full tw-object-center tw-object-cover" %}
 
 {% comment %}
+
+  This template will be rendered if the user selects to customize the header on either a Publication or a Article page.
```

Comment:
**Nitpick** I'd remove references to where a fragment is used from the inside of the fragment itself. This can lead to confusion and gets out of date real quick. If someone was to add hero fragment to another page this comment is not accurate anymore.